src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/config/SecurityConfiguration.java

```diff
@@ -29,15 +30,15 @@ public List<String> getAnonymousPaths() {
         return anonymousPaths;
     }
 
-    @Override
-    public void configure(WebSecurity web) {
-        web.ignoring().mvcMatchers(
-            anonymousPaths.toArray(String[]::new)
-        );
+    @Bean
+    public WebSecurityCustomizer webSecurityCustomizer() {
+        return (web) -> web.ignoring().requestMatchers(anonymousPaths.toArray(String[]::new));
     }
 
-    @Override
-    protected void configure(HttpSecurity http) throws Exception {
+    @Bean
+    @SuppressWarnings("PMD.SignatureDeclareThrowsException")
+    public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
```

Comment:
Is there a more specific Exception we can throw here?

Reply:
This is from HttpSecurity.java, from spring security library Ravi.
public SessionManagementConfigurer<HttpSecurity> sessionManagement() throws Exception {
		return getOrApply(new SessionManagementConfigurer<>());
	}

This method is forcing us to add throws exception to our method.
It didn't give any specific exception Ravi.
So I added General Exception to the signature.

Reply:
ok that's fine then :)